docs/docs-beta/docs/guides/external-systems/non-python.md

```diff
@@ -1,4 +1,141 @@
 ---
 title: "Running non-Python languages"
 sidebar_position: 60
----
\ No newline at end of file
+---
+
+Dagster is written in Python, but that doesn't mean it's limited to running Python to materialize assets. This guide covers how to run some JavaScript with Dagster using Pipes.
+
```

Comment:
Some prerequisites we can include:

- Creating assets in Dagster
- A basic understanding of JavaScript
- Familiarity with Dagster Pipes

Reply:
> Familiarity with Dagster Pipes

may not be a hard prereq? because this guide basically walks the reader through how pipes work?